### PR TITLE
Added IntelliJ IDEA and Android Studio instruction for Autocomplete and Syntax Highlighting

### PR DESCRIPTION
**Pull request by gianluz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,25 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 ```
 
+### Autocomplete and Syntax highlighting in IntelliJ IDEA or Android Studio
+You can activate the autocomplete following this additional steps:
+- Install danger on your local machine
+- Go to `Preferences -> Build, Execution, Deployment -> Compiler -> Kotlin Compiler` (`Preferences -> Kotlin Compiler` in Android Studio)
+- At the bottom you will find a section `Kotlin Scripting`
+- Complete the field `Script template classes` with  `com.danger.kotlin.kts.DangerFileScript`
+- Complete the field `Script templates classpath` with `/usr/local/lib/danger/danger-kotlin-kts.jar:/usr/local/lib/danger/kotlin-main-kts.jar:/usr/local/lib/danger/danger-kotlin.jar`
```
Comment (f-meloni) [inline README.md]: We don't distribute kotlin-main-kts anymore as separate jar I think, and danger-kotlin-kts should not be needed

Comment (gianluz) [reply README.md]: It looks like the autocomplete in intelliJ still not works without it

Comment (f-meloni) [reply README.md]: Without which one of the two?

**Comment by danger-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Any changes to library code should be reflected in the Changelog.

Please consider adding a note there and adhere to the [Changelog Guidelines](https://github.com/Moya/contributors/blob/master/Changelog%20Guidelines.md).
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems">Danger Kotlin</a> against 91b5d342896bd1e873258e2ded7ecca7d7ddb105
</p>

**Pull request merged**